doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Samsung-Artik-5/index.md

```diff
@@ -5,4 +5,117 @@ permalink: /:path/
 sort_idx: 50
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The guide explains how to cross-compile Kaa applications for Samsung Artik 5 based on the Kaa C++ endpoint SDK.
+
+# Configuring Artik 5 board
+
+If this is the first time you use the Artik board, you have to start with configuring your board.
+For this purpose, refer to [the official Artik getting started guide](https://developer.artik.io/documentation/getting-started-beta/).
+After finishing with the guide, you will be able to access your board through both a serial terminal and an ssh client.
+
+# Installing third-party components for C++ SDK
+
+The following third-party components must be installed before building the C++ SDK.
+
+- Boost (1.54 or above).
+- Avro (1.7.5).
+- Botan (only v.1.11) libraries.
```

Comment:
Remove `v.` here or add it for all dependencies. I think we support 1.11 and above